### ISSUE TITLE
Two-column Image

### ISSUE THREAD

**Issue by jona-sassenhagen** [VirtualDataLab/smobsc#6]
@aylinsgl do you still remember this one?

I would like to put an image demonstrating the draw-flip cycle for the Psychopy doc.

**Comment by aylinsgl** [comment]
Ah yes I remember! Will look into it :)

jona-sassenhagen <notifications@github.com> schrieb am Mi., 6. März 2019,
15:54:

**Comment by jona-sassenhagen** [comment]
@aylinsgl :D

**Comment by aylinsgl** [comment]
Something like this?
[image: drawflip.png]

Am Sa., 6. Apr. 2019 um 14:54 Uhr schrieb jona-sassenhagen <
notifications@github.com>:

**Comment by jona-sassenhagen** [comment]
File upload failed!

**Comment by aylinsgl** [comment]
Ups :D

![drawflip](https://user-images.githubusercontent.com/43201604/55730274-17694280-5a18-11e9-9803-00b39feb1e37.png)

**Comment by jona-sassenhagen** [comment]
Ja, in etwa. Ich würde es eher unterteilen in code, front, back. Vielleicht front und back mit Monitor im Hintergrund oder so?

**Comment by aylinsgl** [comment]
![drawflip2 0](https://user-images.githubusercontent.com/43201604/55746305-a6d31d80-5a39-11e9-949d-d40d6b7796f9.png)

like this maybe?

**Comment by aylinsgl** [comment]
oder lieber auf der frontside den text mittig weil er erscheint ja nicht unten auf dem bildschirm eigentlich

**Comment by aylinsgl** [comment]
![draw_flip_3 0](https://user-images.githubusercontent.com/43201604/56284555-0f468c80-6115-11e9-96a8-79ee3d186782.png)

**Comment by jona-sassenhagen** [comment]
Hm, ok - aber vielleicht auf zwei Bildschirmen, oder der Backside screen hinter dem Monitor?

Und ich würde tatsächlich es stage bei stage zeigen. Also ein Bild für Front leer, Back "This is what drawing text looks like", ein Bildschirm Front "This is what drawing text looks like", back leer, dann noch ein screen .. Und daneben jeweils die entsprechende Codezeile.

Und dannnnnnnnnn bin ich zu Frieden :D

**Comment by aylinsgl** [comment]
havent forgotten about this will work on it again soon ... :D
